src/app/clients/edit/[id]/autofill.ts

```diff
@@ -35,17 +35,19 @@ const autofill = (
             };
         });
 
-    const { nappySize, extraInformation } = processExtraInformation(clientData.extra_information);
+    const { nappySize, extraInformation } = processExtraInformation(
+        clientData.extra_information ?? ""
+    );
 
     const noPostcode = clientData.address_postcode === null;
 
     return {
-        fullName: clientData.full_name,
-        phoneNumber: clientData.phone_number,
-        addressLine1: noPostcode ? "" : clientData.address_1,
-        addressLine2: noPostcode ? "" : clientData.address_2,
-        addressTown: noPostcode ? "" : clientData.address_town,
-        addressCounty: noPostcode ? "" : clientData.address_county,
+        fullName: clientData.full_name ?? "",
+        phoneNumber: clientData.phone_number ?? "",
+        addressLine1: noPostcode ? "" : clientData.address_1 ?? "",
+        addressLine2: noPostcode ? "" : clientData.address_2 ?? "",
+        addressTown: noPostcode ? "" : clientData.address_town ?? "",
+        addressCounty: noPostcode ? "" : clientData.address_county ?? "",
```

Comment:
I don't think I quite understand what you're asking about. In this PR, the other address fields became nullable. Therefore we must use `clientData.address_1 ?? ""` instead of `clientData.address_1` so that we have a string to display. That's the change I made, but please let me know if I misunderstood and you're asking about the already existing code. :)

Reply:
Yeah the line that I have in mind is `const noPostcode = clientData.address_postcode === null;` - my assumption is that when you nullify the address, you also nullify the postcode, so it would never reach `clientData.address_xyz` 

Guess then the worry is, are there cases where we have a postcode but no address? Old logic implies no (as you only added the `?? ""` for this case of deleting the user), but it might have been wrong in the first place

Hopefully that makes more sense?

Reply:
Does this answer your question? :)

|postcode → address_wxyz ↓ | Non-null | Null |
|--------|--------|--------|
| Non-null | Active client with address | Active NFA client|
| Null | Not possible, but DB schema doesn't realise that | Client has been deleted through front end|

Reply:
Think I looked at this before the other comment about NFA clients, yep makes sense now, thanks :)